### PR TITLE
Margarits branch

### PR DESCRIPTION
**Pull request by margaritluch**
everything working, corrected design of my pug files

### CODE REVIEW

File: models/workoutProgram.js [hunk 1]
```diff
@@ -13,11 +14,13 @@ const workoutProgramSchema = mongoose.Schema({
...
     required: true,
     trim: true
-  }
-});
-
-
+  },
 
+  excercises:
+    /*{type: mongoose.Schema.Types.ObjectId, ref: 'excerciseSchema'} */
```
Comment (Drsela) [inline models/workoutProgram.js]: Remove commented code :)

File: views/excercises.pug [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            th Description
+            th Set
+            th Duration
+            each excercise in ex_array
+                tr
+                  td #{excercise.nameOfExcercise}
+                  td #{excercise.descriptionOfExcercise}
+                  td #{excercise.setOfExcercise}
+                  td #{excercise.durationOfExcercise}
+        div
```
Comment (Drsela) [inline views/excercises.pug]: I think this div could be removed

Comment (margaritluch) [reply views/excercises.pug]: ok

File: models/workoutProgram.js (hunk 1 above)
Comment (margaritluch) [reply models/workoutProgram.js]: ok

**Comment by margaritluch** [comment]
aah common Emma, det er ikke en krav at have minuter med...

**Pull request merged**